### PR TITLE
AMP-30535 created patch to fix module visibility for parent

### PR DESCRIPTION
**Pull request by timothygachengo**

### CODE REVIEW

File: amp/.DS_Store
Comment (jdeanquin-dg) [inline amp/.DS_Store:1]: Please don't commit this file. Add it to .gitignore

File: amp/deployConfigs/selected.properties
Comment (jdeanquin-dg) [inline amp/deployConfigs/selected.properties:1]: Please do not commit this file, so we don't generate conflicts with other branches

File: amp/WEB-INF/src/org/digijava/module/aim/dbentity/AmpModulesVisibility.java
Comment (jdeanquin-dg) [inline amp/WEB-INF/src/org/digijava/module/aim/dbentity/AmpModulesVisibility.java:1]: Even though the code committed is better, it's out of the scope of this change, so please revert.

**Review by jdeanquin-dg — CHANGES_REQUESTED** [review]

File: amp/deployConfigs/selected.properties
Comment (timothygachengo) [reply amp/deployConfigs/selected.properties:1]: Should I add this to the .gitignore?

Comment (jdeanquin-dg) [inline amp/deployConfigs/selected.properties:1]: you dont need to delete, you dont need to commit please revert the deletion and remove it from gitignore

**Review by jdeanquin-dg — CHANGES_REQUESTED** [review]

**Review by jdeanquin-dg — DISMISSED** [review]

**Comment by jdeanquin-dg** [comment]
replaced by #4137

**Pull request closed without merging**